app/medication/edit/controller.js

```diff
@@ -69,22 +70,24 @@ export default AbstractEditController.extend(InventorySelection, FulfillRequest,
   updateCapability: 'add_medication',
 
   afterUpdate: function() {
-    var alertTitle = 'Medication Request Saved',
-      alertMessage = 'The medication record has been saved.',
+    let i18n = this.get('i18n');
+    var alertTitle,
+      alertMessage,
       isFulfilled = this.get('isFulfilled');
     if (isFulfilled) {
-      alertTitle = 'Medication Request Fulfilled';
+      alertTitle = i18n.t('medication.alerts.fulfilled_title');
       alertMessage = 'The medication request has been fulfilled.';
       this.set('model.selectPatient', false);
     } else {
-      alertTitle = 'Medication Request Saved';
-      alertMessage = 'The medication record has been saved.';
+      alertTitle = i18n.t('medication.alerts.saved_title');
+      alertMessage = i18n.t('medication.alerts.saved_message');
     }
     this.saveVisitIfNeeded(alertTitle, alertMessage);
   },
 
   _addNewPatient: function() {
-    this.displayAlert('Please Wait', 'A new patient needs to be created...Please wait..');
+    let i18n = this.get('i18n');
+    this.displayAlert(i18n.t('common.please_wait'), i18n.t('messages.new_patient_has_to_be_created'));
```

Comment:
@alexpelan again common isn't nested - I'm assuming your intention was to not nest it?